dashboard/app/serializers/api/v1/pd/application_serializer.rb

```diff
@@ -139,4 +141,16 @@ def school_stats
       two_or_more_races_percent: percent_string(stats.student_tr_count, stats.students_total)
     }
   end
+
+  def scholarship_status
+    object.try(:scholarship_status)
+  end
+
+  def scholarship_dropdown_options
```

Comment:
I don't think this list of options belongs in the serializer. This is the same for every application, and shouldn't be something we serialize every single time.

Instead this map should be in Shared Constants